server/template.go

```diff
@@ -188,10 +199,10 @@ func init() {
 
 	template.Must(masterTemplate.New("pullRequestReviewNotification").Funcs(funcMap).Parse(`
 {{template "user" .GetSender}}
-{{- if eq .GetReview.GetState "approved" }} approved your pull request
-{{- else if eq .GetReview.GetState "changes_requested" }} requested changes on your pull request
-{{- else if eq .GetReview.GetState "commented" }} commented on your pull request
-{{- end }} {{template "eventRepoPullRequest" .}}
+{{- if eq .GetReview.GetState "approved" }} approved your pull request {{template "eventRepoPullRequest" .}}
+{{- else if eq .GetReview.GetState "changes_requested" }} requested changes on your pull request {{template "eventRepoPullRequest" .}}
```

Comment:
Is it possible for `approved` and `changes_requested` events to have a comment link attached to them? For example, if someone requests changes, is it possible to link to the main comment of the requested changes? Or would that be handled by the `commented` event simultaneously?

Reply:
I think these events different from `commented`. since we have the review object this should be possible only if the . I will check with github webhook what kind of data they provide on these events

anyway that will improve the template code defined as `pullRequestReviewNotification`

Reply:
Sounds good, this is looking great!